README.md

```diff
@@ -9,28 +9,36 @@ Workflow:
 
 ## Getting started
 
-### Install
+### Run via Docker
+
+Build the image:
 
 ```bash
->> go get -t ./...
+>> docker build -t robokache .
 ```
 
-### Run
+Run the image: 
 
 ```bash
->> go run ./cmd
+>> docker run -it -p 8080:8080 robokache
 ```
 
 * Got to <http://lvh.me:8080/>
 * Copy ID token from developer tools into authentication field
 * Have fun
 
-### Test
+### Test via Docker
```

Comment:
Running the tests in Docker feels odd to me because I generally imagine Docker as a deployment tool. I figure if someone needs to run the tests, they're developing, and if they're developing they're probably doing it outside of Docker.

Reply:
I personally use Docker for a development environment as well as a deployment tool. One advantage of this is that there is no need to install Go for development. I'd be happy to revert the changes to the testing section of the README if this is not the approach we would like to use.

Reply:
My impression is that using Docker for development is not common practice, so let's imagine the typical visitor to this repo is developing with a native Go install, and those who want to use Docker for development know how to adapt.

Reply:
I updated the README to use the local Go installation for development.